integration-test/583-merge-landuse.py

```diff
@@ -648,8 +648,13 @@ def _freeze(thing):
     feature_props = set()
 
     for feature in landuse:
-        # ignore ID, area when checking for uniqueness
         p = feature['properties'].copy()
+
+        # ignore label placements
+        if p.get('label_placement'):
```

Comment:
Ah, good point! It usually is truthy, perhaps even always, but I like your spelling better in case it's not.